pkg/scheduler/preemption/preemption.go

```diff
@@ -128,40 +128,31 @@ func (p *Preemptor) issuePreemptions(ctx context.Context, targets []*workload.In
 	defer cancel()
 	workqueue.ParallelizeUntil(ctx, parallelPreemptions, len(targets), func(i int) {
 		target := targets[i]
-		patch := workload.BaseSSAWorkload(target.Obj)
-		patch.Status.Conditions = []metav1.Condition{{
-			Type:               kueue.WorkloadAdmitted,
-			Status:             metav1.ConditionFalse,
-			LastTransitionTime: metav1.Now(),
-			Reason:             "Preempted",
-			Message:            "Preempted to accommodate a higher priority Workload",
-		}}
-		err := p.applyPreemption(ctx, patch)
-		if err != nil {
-			errCh.SendErrorWithCancel(err, cancel)
-			return
-		}
-
-		workload.SetEvictedCondition(patch, kueue.WorkloadEvictedByPreemption, "Preempted to accommodate a higher priority Workload")
-		err = workload.ApplyAdmissionStatus(ctx, p.client, patch, false)
-		if err != nil {
-			errCh.SendErrorWithCancel(err, cancel)
-			return
-		}
+		if !apimeta.IsStatusConditionTrue(target.Obj.Status.Conditions, kueue.WorkloadEvicted) {
```

Comment:
Is it intentional to use `!apimeta.IsStatusConditionTrue` instead of `apimeta.IsStatusConditionFalse`?

Reply:
That sounds good.